src/services/message.service.ts

```diff
@@ -44,15 +44,20 @@ export class MessageService {
   async sendReactiveMessage(
     message: IMessage,
     embedData: IEmbedData,
-    lambda: Function
+    lambda: Function,
+    options: IReactionOptions
   ): Promise<IMessage> {
     const msg: IMessage = await message.reply(embedData.embeddedMessage);
+    const minEmotes: number = embedData.emojiData.length - (options.reactionCutoff || 1);
+
     await Promise.all(embedData.emojiData.map((reaction) => msg.react(reaction.emoji)));
+    msg.react('❎'); // Makes cancel available on all reactions (We coudl also make it an option in the future)
```

Comment:
could*
You should install [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker). I love it